app/services/induction/transfer_and_continue_existing_programme.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+class Induction::TransferAndContinueExistingProgramme < BaseService
```

Comment:
This service class is really similar to a couple of them for schools to transfer participants. Cannot we merge the 3 in just one or maybe 2?